### ISSUE TITLE
[Feature]: MapTool theme and stat sheet CSS improvements

### ISSUE THREAD

**Issue by Baaaaaz** [RPTools/maptool#5961]
### Describe the Problem

Currently unable to style frame/dialog/overlay/chat content according to specific user color/accessibility preferences.

Also some html elements are not handled in the current MapTool theme files (other buttons types, other form control elements, etc), so look to improve style coverage.

### The Solution you'd like

Incorporate color/accessibility preferences into the generated MapTool theme and stat sheet CSS files, along with other CSS improvements for styling html elements similar to MapTool UI.

### Alternatives that you've considered.

Expose certain user styling preferences via `getInfo("Client")`, including map label colors, arrow colors, if a dark theme is selected, etc so frames/dialogs/overlays/chat outputs can be styled individually for each user.  Some user preferences are already exposed via `getInfo("Client")` - attempted in [[PR](https://github.com/RPTools/maptool/pull/5962)] but a better way was suggested, now added as the solution I'd like above.
